src/views/viewer/DiscoverView.tsx

```diff
@@ -0,0 +1,235 @@
+import styled from '@emotion/styled'
+import React from 'react'
+
+import { useVideosConnection } from '@/api/hooks'
+import { LimitedWidthContainer } from '@/components/LimitedWidthContainer'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { GridItem } from '@/shared/components/LayoutGrid'
+import { Text } from '@/shared/components/Text'
+import { FeaturedVideoCategoryCard, VideoCategoryCard } from '@/shared/components/VideoCategoryCard'
+import { SvgVideoCategoriesAutosAndVehicles } from '@/shared/icons/VideoCategoriesAutosAndVehicles'
+import { SvgVideoCategoriesComedy } from '@/shared/icons/VideoCategoriesComedy'
+import { SvgVideoCategoriesEducation } from '@/shared/icons/VideoCategoriesEducation'
+import { SvgVideoCategoriesEntertainment } from '@/shared/icons/VideoCategoriesEntertainment'
+import { SvgVideoCategoriesFilmAndAnimation } from '@/shared/icons/VideoCategoriesFilmAndAnimation'
+import { SvgVideoCategoriesGaming } from '@/shared/icons/VideoCategoriesGaming'
+import { SvgVideoCategoriesHowtoAndStyle } from '@/shared/icons/VideoCategoriesHowtoAndStyle'
+import { SvgVideoCategoriesMusic } from '@/shared/icons/VideoCategoriesMusic'
+import { SvgVideoCategoriesNewsAndPolitics } from '@/shared/icons/VideoCategoriesNewsAndPolitics'
+import { SvgVideoCategoriesNonprofitsAndActivism } from '@/shared/icons/VideoCategoriesNonprofitsAndActivism'
+import { SvgVideoCategoriesPeopleAndBlogs } from '@/shared/icons/VideoCategoriesPeopleAndBlogs'
+import { SvgVideoCategoriesPetsAndAnimals } from '@/shared/icons/VideoCategoriesPetsAndAnimals'
+import { SvgVideoCategoriesScienceAndTechnology } from '@/shared/icons/VideoCategoriesScienceAndTechnology'
+import { SvgVideoCategoriesSports } from '@/shared/icons/VideoCategoriesSports'
+import { SvgVideoCategoriesTravelAndEvents } from '@/shared/icons/VideoCategoriesTravelAndEvents'
```

Comment:
Also, why do you import icons like that? I don't see all these icons in `shared/icons/index`, why you didn't add that? Just run: `yarn codegen:svgs && yarn prettier . --write` and you should be able to do: 
```tsx
import {
  SvgVideoCategoriesAutosAndVehicles,
  SvgVideoCategoriesComedy
   ...
} from '@/shared/icons'
```